### PR TITLE
Task/hotfix 1.0.1 merge

### PR DESCRIPTION
**Pull request by wtritch**
This PR merges hotfix 1.0.1 back into "develop."

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -37,7 +37,7 @@ String dockerPrefix = "smarterbalanced"
 
 allprojects {
     group = 'org.opentestsystem.rdw.reporting'
-    version = '1.0.0-SNAPSHOT'
+    version = '1.0.1-SNAPSHOT'
```
Comment (malaffoon) [inline build.gradle:40]: personally i'd change this to 1.1.0 but i realize you are just merging changes

**Comment by malaffoon** [comment]
Are you planning to move the shared code into RDW_Common security at some point? Or should we just create a JIRA for it?

File: build.gradle (hunk 1 above)
Comment (wtritch) [reply build.gradle:40]: I meant to bring that up, our 'develop' versions seem to be a little behind.  I don't think this is the PR to fix it/them, but something to be aware of.

**Review by malaffoon — APPROVED** [review]

**Comment by wtritch** [comment]
I don't want to merge it into RDW_Common, because that will add a dependency on SAML to RDW_Common.  I'd rather merge it into our upcoming RDW_Reporting/web-base or whatever module that will be shared by admin and reporting webapps

**Pull request merged**